stable/polaris/templates/dashboard.deployment.yaml

```diff
@@ -112,9 +112,13 @@ spec:
       {{- end }}
       serviceAccountName: {{ template "polaris.serviceAccountName" . }}
       nodeSelector:
+      {{- if .Values.dashboard.nodeSelector }}
       {{- with .Values.dashboard.nodeSelector }}
 {{ toYaml . | indent 8 }}
-        {{- end }}
+      {{- end }}
+      {{- else }}
+        kubernetes.io/os: linux
```

Comment:
@sudermanjr does this seem good to you as a default label? Is it something we should adopt across all our charts?